### DISCUSSION TITLE
Milvus Clojure Library

### DISCUSSION THREAD

**Discussion by todd-kim** [milvus-io/milvus#32239] [Announcement]
Hello, I’ve created a library to make Milvus easier to use in clojure. Although it’s a wrapper around the Milvus Java SDK, it should be easy to use from clojure. It doesn’t support all of Milvus’ APIs yet, but I’ve created all the important ones. https://github.com/constacts/milvus-clj

**Comment by xiaofan-luan** [comment]
good to know that.
Every thing goes well so far?

**Reply by todd-kim** [reply]
It looks like it's ready for production. I'm excited to contribute to the Milvus ecosystem.
